test/src/command_runner_test.dart

```diff
@@ -16,6 +16,12 @@ class MockPubUpdater extends Mock implements PubUpdater {}
 
 class MockProgress extends Mock implements Progress {}
 
+class MockDirectory extends Mock implements Directory {}
+
+class MockFile extends Mock implements File {}
+
+class MockStdout extends Mock implements Stdout {}
```

Comment:
**Nit:** Can we make them private?

Reply:
I think we should have a follow PR to do that on all tests then because right now it is a mix of both.

Reply:
Resolving this conversation, I made an issue to track this change: https://github.com/VeryGoodOpenSource/very_good_cli/issues/827

Reply:
Now al Mocks are private, can we update these mocks?